src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java

```diff
@@ -29,15 +32,17 @@ public void EditAccount_001_closeAccount() {
     editAccount.verifyAccountClosedMessage();
   }
 
-  @Test(groups = "EditAccountTest", dependsOnMethods = "EditAccount_001_closeAccount")
+  @Test(dependsOnMethods = "EditAccount_001_closeAccount")
   public void EditAccount_002_verifyAccountClosed() {
     WikiBasePageObject base = new WikiBasePageObject();
-    SpecialUserLoginPageObject login = base.openSpecialUserLoginOnWiki(wikiURL);
-    login.login(credentials.userNameClosedAccount, credentials.passwordClosedAccount);
-    login.verifyClosedAccountMessage();
+    SpecialUserLoginPageObject login = new SpecialUserLoginPageObject(driver);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "login" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)